samples/secret.yaml

```diff
@@ -33,10 +33,35 @@
   # Default value: ""
   # This is an optional field from v2.10.0 onwards for PowerFlex storage system >=4.0.x
   nasName: "nas-server"
-# # To add more PowerFlex systems, uncomment the following lines and provide the required values
+  # zone is a cluster availability zone to which the PowerFlex system should be bound.
+  # The mapping is one-to-one - the PowerFlex system cannot belong to more than one zone.
+  # Ideally, the PowerFlex system and cluster nodes that define the availability zone would be
+  # geographically co-located.
+  # Optional: true
+  # Default value: none
+  zone:
+    # The name of the container orchestrator's availability zone to which the PowerFlex system
+    # should be mapped.
+    name: "zoneA"
+    # A list of protection domains to be utilized by PowerFlex system with optional pool assignment.
+    # Users will need to create these definitions in the PowerFlex system prior to assigning their
+    # values here.
+    domains:
+      - name: "domain1"
+        # pool:
+        # Optional: true
+        # Default value: ""
+        pool: "pool1"
+# To add more PowerFlex systems, uncomment the following lines and provide the required values
 # - username: "admin"
 #   password: "password"
 #   systemID: "2b11bb111111bb1b"
 #   endpoint: "https://127.0.0.2"
 #   skipCertificateValidation: true
 #   mdm: "10.0.0.3,10.0.0.4"
+#   zone:
+#     name: "ZoneB"
+#     domains:
+#       - name: "domain001"
+#       - name: "domain002"
```

Comment:
what is this saying, 2 domains with the same pool name pool002?

Reply:
`domains` is actually a list of items, with an item being comprised of name and optional pool.
so the first domain does not have a pool selected and the second does.
Just trying to illustrate the available options to the user.